lizmap/modules/lizmap/classes/qgisForm.class.php

```diff
@@ -254,6 +254,20 @@ public function __construct($layer, $form, $featureId, $loginFilteredOverride)
             $this->formControls[$fieldName] = $formControl;
         }
 
+        // Deactivate undisplayed fields in Drag and Drop form
+        $attributeEditorForm = $qgisForm->getAttributesEditorForm();
```

Comment:
![image](https://user-images.githubusercontent.com/2145040/104210802-94b6b800-5433-11eb-9dc9-95e4d9a6463b.png)

Do you mean `$this->getAttributesEditorForm()`?

If I change to `$this` then in console I have when clicking edition in attribute table : 
![image](https://user-images.githubusercontent.com/2145040/104211233-1e668580-5434-11eb-8dd1-b3204285348b.png)

Please use https://github.com/3liz/lizmap-web-client/commit/00dbf6a521673f94e81226e4f3d63e9d96ead5dd to test.

Reply:
Changes and test done